aocr/util/data_augmentation.py

```diff
@@ -0,0 +1,199 @@
+import random
+from typing import Tuple
+
+import cv2
+import numpy as np
+from PIL import Image, ImageEnhance, ImageOps
+
+
+__all__ = ['add_random_lines',
+           'add_random_padding',
+           'crop_image',
+           'modify_brightness',
+           'modify_contrast',
+           'random_resize',
+           'random_rotation',
+           'modify_sharpness'
+           ]
+
+
+def add_random_lines(img: Image.Image, num_lines_range: Tuple[int, int] = (1, 3)) -> Image.Image:
+    '''
+    Add random lines mimicking scanning artifacts.
+
+    Args:
+        img: Image to modify
+        num_lines_range: Minimum and maximum number of lines to be added
+
+    Returns:
+        A modified Pillow Image.
+    '''
```

Comment:
Incredibly minor, but the convention for docstrings is double-quotes 😄 
```suggestion
    """
    Add random lines mimicking scanning artifacts.

    Args:
        img: Image to modify
        num_lines_range: Minimum and maximum number of lines to be added

    Returns:
        A modified Pillow Image.
    """
```